tests/integrations/celery/test_celery.py

```diff
@@ -255,18 +256,14 @@ def test_no_stackoverflows(celery):
 
     @celery.task(name="dummy_task")
     def dummy_task():
-        with configure_scope() as scope:
-            scope.set_tag("foo", "bar")
-
+        Scope.get_isolation_scope().set_tag("foo", "bar")
         results.append(42)
 
     for _ in range(10000):
         dummy_task.delay()
 
     assert results == [42] * 10000
-
-    with configure_scope() as scope:
-        assert not scope._tags
+    assert not Scope.get_current_scope()._tags
```

Comment:
Should we be checking this on current scope?

Reply:
Oops, yeah this should probably be `get_isolation_scope`